src/smart-components/portfolio/portfolio-products.js

```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Route } from 'react-router-dom';
+
+import ToolbarRenderer from '../../toolbar/toolbar-renderer';
+import createPortfolioToolbarSchema from '../../toolbar/schemas/portfolio-toolbar.schema';
+import AddPortfolioModal from './add-portfolio-modal';
+import RemovePortfolioModal from './remove-portfolio-modal';
+import SharePortfolioModal from './share-portfolio-modal';
+import OrderModal from '../common/order-modal';
+import PortfolioEmptyState from './portfolio-empty-state';
+import ContentGallery from '../content-gallery/content-gallery';
+
+const PortolioProducts = ({
```

Comment:
I think we should try to use the 'portfolio item' term, not 'product' everywhere possible - except the strings that are actually displayed on the screen.